recipe/bld.bat

```diff
@@ -14,7 +14,7 @@ cmake .. -G "Ninja" ^
     -DTTK_ENABLE_CPU_OPTIMIZATION=OFF ^
     -DTTK_ENABLE_KAMIKAZE=OFF ^
     -DTTK_ENABLE_MPI=OFF ^
-    -DTTK_ENABLE_OPENMP=ON
+    -DTTK_ENABLE_OPENMP=OFF
```

Comment:
sorry I forgot to mention that we indeed recommend to disable OpenMP on windows (https://topology-tool-kit.github.io/installation-windows.html). ttk makes many OpenMP 2 calls (which are not supported by msvc).

Reply:
Thanks for your message! I was confused, because it actually builds just fine with vs2019, but not with vs2015.